Src/Plugins/Tests.Security/Validators/SEC101_036.MySqlConnectionStringValidatorTests.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+
+using Microsoft.CodeAnalysis.Sarif.PatternMatcher.Sdk;
+
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Sarif.PatternMatcher.Plugins.Security.Validators
+{
+    public class MySqlConnectionStringValidatorTests
+    {
+        [Fact]
+        public void MySqlConnectionStringValidator_Test()
+        {
+            string fingerprintText = "";
+            if (string.IsNullOrEmpty(fingerprintText))
+            {
+                return;
+            }
+
+            var fingerprint = new Fingerprint(fingerprintText);
+            string message = null;
```

Comment:
I would have put this line above, in order to improve readability, small style point.